### PR TITLE
witch hat costume for halloween party cat 🎃

### PR DESCRIPTION
**Pull request by leozamboni**
a witch hat for halloween but one day later :(

### CODE REVIEW

**Comment by flowln** [comment]
You'd probably coordinate with the folks at #372 and also add a spooky Rory <:)

**Comment by AshtakaOOf** [comment]
Should I make a Spooky Rory?

**Comment by leozamboni** [comment]
please, themed stuff are cool

**Comment by Scrumplex** [comment]
Please integrate this with the system introduced by #372

**Comment by santiagocezar** [comment]
Basically you just need to add a `-ween` suffix (or anything else) to the cat string in the logic and save the variants with that suffix too. So most of the logic can be reused!

**Comment by RaptaG** [comment]
Why’s this closed?

**Comment by leozamboni** [comment]
I'm fixing

**Comment by Scrumplex** [comment]
In that case making this a draft is nicer I think

**Comment by Scrumplex** [comment]
Oh. I thought I can make it a draft myself

**Comment by leozamboni** [comment]
github automatically closed the pr to update my fork, I don't know why

**Comment by TheKodeToad** [comment]
It seems the if statement's formatting is a little inconsistent.
Yes...
That's right...
Time for clang-format.

**Comment by leozamboni** [comment]
done :)

**Comment by leozamboni** [comment]
- [6b856c9](https://github.com/PrismLauncher/PrismLauncher/pull/373/commits/6b856c93eabb6358bff84b6fc0c31415476d3b9b) formatted the entire file by mistake
- [498a1cd](https://github.com/PrismLauncher/PrismLauncher/pull/373/commits/498a1cdf1693710d17af8406fb95789f8be2ea89) revert and format only the block

File: result [hunk 1]
```diff
@@ -0,0 +1 @@
+/nix/store/ps23952qj1qlvi7vm0jf2n03i9mlx8fq-prismlauncher-20221102
```
Comment (santiagocezar) [inline result]: I don't think this should be here?

**Comment by santiagocezar** [comment]
BTW you can probably just squash the changes once you're done, so don't worry about those commits

**Review by santiagocezar — APPROVED** [review]
image

hat cat 👍

**Review by AshtakaOOf — APPROVED** [review]
Cat are cool so I approve :+1:

**Review by Scrumplex — APPROVED** [review]
LGTM

**Comment by Scrumplex** [comment]
@leozamboni Can you squash all your commits into one? We could do that when merging on GitHub, but that would replace your own signature with GitHub's signature. If you need help, do not hesitate to ask.

**Pull request closed without merging**